code/qcommon/q_shared.h

```diff
@@ -445,9 +445,7 @@ qboolean Q_IsColorString(const char *p); // ^[0-9a-zA-Z]
 #define S_COLOR_WHITE "^7"
 
 extern vec4_t g_color_table[8];
-extern vec4_t spraycolors[];
-//#define NUM_SPRAYCOLORS ( sizeof( spraycolors[] ) / sizeof( spraycolors[0] ) )
-#define NUM_SPRAYCOLORS 6
+#define NUM_SPRAYCOLORS 8
```

Comment:
maybe move this define over `g_color_table` and use `extern vec4_t g_color_table[NUM_SPRAYCOLORS];` (and maybe even rename `NUM_SPRAYCOLORS` to `NUM_COLORS`)